pybit/_websocket_stream.py

```diff
@@ -144,21 +192,22 @@ def resubscribe_to_topics():
         while (
             infinitely_reconnect or retries > 0
         ) and not self.is_connected():
-            logger.info(f"WebSocket {self.ws_name} attempting connection...")
+            logger.info("WebSocket %s attempting connection...", self.ws_name)
             self.ws = websocket.WebSocketApp(
                 url=url,
-                on_message=lambda ws, msg: self._on_message(msg),
-                on_close=lambda ws, *args: self._on_close(),
-                on_open=lambda ws, *args: self._on_open(),
-                on_error=lambda ws, err: self._on_error(err),
-                on_pong=lambda ws, *args: self._on_pong(),
+                on_message=self._on_message,
+                on_close=self._on_close,
+                on_open=self._on_open,
```

Comment:
Lambdas were replaced with named functions because of logging purposes.
So now we'll have:
```
2024-06-20 19:20:54,291 - websocket - ERROR - error from callback <bound method _WebSocketManager._on_message of <pybit.unified_trading.WebSocket object at 0x7fc0a983fc40>>: 'kline.1.FOGEUSDT'
2024-06-20 19:20:54,482 - websocket - ERROR - error from callback <bound method _WebSocketManager._on_error of <pybit.unified_trading.WebSocket object at 0x7fc0a983fc40>>: 'kline.1.FOGEUSDT'
```
instead of:
```
2024-06-20 09:50:05,306 - websocket - ERROR - error from callback <function _WebSocketManager._connect.<locals>.<lambda> at 0x7f7b3854a8c0>: Expecting ':' delimiter: line 1 column 35 (char 34)
2024-06-20 09:50:05,503 - websocket - ERROR - error from callback <function _WebSocketManager._connect.<locals>.<lambda> at 0x7f7b38429a20>: Expecting ':' delimiter: line 1 column 35 (char 34)
```
We'll se the function the exception was raised from, instead of `<locals>.<lambda>`